CHANGELOG.md

```diff
@@ -1,5 +1,101 @@
+* add KsqlQueryStatus to decouple from KafkaStreams.State ([#5029](https://github.com/confluentinc/ksql/pull/5029)) ([e8cbcde](https://github.com/confluentinc/ksql/commit/e8cbcde548e1f3078b1396173a3f55f86ee20626))
+* support multi-way joins ([#5064](https://github.com/confluentinc/ksql/pull/5064)) ([64dd39e](https://github.com/confluentinc/ksql/commit/64dd39ee77928221f08371d5e4424839d20c09f6))
+* Adds rate limiting to pull queries ([#4951](https://github.com/confluentinc/ksql/pull/4951)) ([6284111](https://github.com/confluentinc/ksql/commit/6284111652252b63cfc233bdfc3e08dbff983bbd))
+* CORS support for Vert.x based ksqlDB ([#4944](https://github.com/confluentinc/ksql/pull/4944)) ([5632893](https://github.com/confluentinc/ksql/commit/56328931d7eb26699fb5bd0940cfd03f7f9e53f9))
+* Do not allow access to new streaming endpoints using HTTP1.x ([#5193](https://github.com/confluentinc/ksql/pull/5193)) ([8b90035](https://github.com/confluentinc/ksql/commit/8b90035c6facf47d81dfd1616784b191141dce31))
+* fail startup if command contains incompatible version ([#5104](https://github.com/confluentinc/ksql/pull/5104)) ([a1751b1](https://github.com/confluentinc/ksql/commit/a1751b1689532ef9abd436d5f27fe9aa8ff555ac))
+* Implement cluster health check endpoints ([#5126](https://github.com/confluentinc/ksql/pull/5126)) ([fbc9bf0](https://github.com/confluentinc/ksql/commit/fbc9bf019ec53b02af11832e12ed944ae4e424d0))
+* klip-14 - rowtime as pseduocolumn ([#5150](https://github.com/confluentinc/ksql/pull/5150)) ([d541420](https://github.com/confluentinc/ksql/commit/d541420acf61f0e0e7b25d91120886007dbdae01))
+* Migrate /ksql and /ksql/terminate endpoints ([#4851](https://github.com/confluentinc/ksql/pull/4851)) ([af6c792](https://github.com/confluentinc/ksql/commit/af6c7922d3c1b0afdd87e9a079e6f95be9780172))
+* scatter gather query status from all servers in cluster for 'SHOW QUERIES [EXTENDED]' statement ([#4875](https://github.com/confluentinc/ksql/pull/4875)) ([7385a31](https://github.com/confluentinc/ksql/commit/7385a31dd33293b76e594649213da2abb81b8ddf))
+* speed up restarts by not building topologies for terminated queries ([#5002](https://github.com/confluentinc/ksql/pull/5002)) ([2472382](https://github.com/confluentinc/ksql/commit/24723824635e7ca175bc5be1fbf6a415010b00bb))
+* Support wildcards for unauthed paths ([#4945](https://github.com/confluentinc/ksql/pull/4945)) ([9173f1d](https://github.com/confluentinc/ksql/commit/9173f1dbf92fea0f1b17b6c85fb8a6c25e789ce3))
+* transient queries added to show queries output ([#5105](https://github.com/confluentinc/ksql/pull/5105)) ([e8a2a63](https://github.com/confluentinc/ksql/commit/e8a2a63210219cbac7d979ec0915f1d2016398e2))
+
+
+### Bug Fixes
+
+* 'drop (stream|table) if exists' fails if source does not exist ([#4872](https://github.com/confluentinc/ksql/pull/4872)) ([b0669a0](https://github.com/confluentinc/ksql/commit/b0669a044b53407fba3ceb11bddee73e69c0006f))
```

Comment:
Can we reword this to "'drop (stream|table) if exists' should not fail if source does not exist"? Otherwise it makes it sound like the change was to make it start failing, which is the opposite of what the fix actually does.

Reply:
done